Patrice/patrice_stocks.py

```diff
@@ -0,0 +1,61 @@
+print("Challenge 3.2: Playing with the stock market")
+
+print()
+
+# Creating variables to store the current (approximate) market price of these 5 companies - Amazon, Apple, Facebook, Google and Microsoft.
+
+amazon = 3000
+apple = 100
+fb = 250
+google = 1400
+msft = 200
+
+print("Challenge 3.2.1: Taking user input")
+# TODO: Write code to ask the client his name and save it to a variable.
+name = input( "enter your name?")
+print(name)
+
+
+# TODO: Write code to ask the client his savings and save it to another variable.
+savings = input("enter your savings")
+print(savings)
+
+# TODO: Write code to ask the client the stock he is interested in and save it to another variable, as shown below.
+
+stock = input("Which stock are you interested in? Type 'amzn' for Amazon, 'aapl' for Apple, 'fb' for Facebook, 'goog' for Google and 'msft' for Microsoft.")
+print(stock)
+
+print("Challenge 3.2.2: Perform user-specific calculations")
+# TODO: You have all 3 user inputs stores in variables. Based on that, write conditional (if-elif-else) statements to find out the number of stocks of the company that can be purchased with the savings amount.
+if stock == "amazon":
```

Comment:
Follow this format for your conditional statements:
`if stock == 'amzn':
    price = amazon
    shares = savings /  price
      `